libs/limits-and-boundaries/src/lib/components/MarksAndStyle.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { SliderMarks } from 'antd/es/slider';
+
+const style: React.CSSProperties = {
+  marginLeft: 70,
+  marginRight: 70,
+  marginTop: 20,
+};
+
+const marksTemp: SliderMarks = {
+  0: {
+    label: '0°C',
+  },
+  50: {
+    label: '50°C',
+  },
+  100: {
+    label: '100°C',
+  },
+  150: {
+    label: '150°C'
+  }
+};
+
+const marksHum: SliderMarks = {
+  0: {
+    label: '0%',
+  },
+  100: {
+    label: '100%',
+  },
+};
+
+const marksCO2: SliderMarks = {
+  0: {
+    label: '0%',
+  },
+  100: {
+    label: '100%',
+  },
+};
+
+export { marksTemp };
```

Comment:
You can export the constants directly

`export const marksCO2: SliderMarks = {`